acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -69,23 +143,19 @@ def rpc_blocking_request(broker, targets,
   # Event machine is required by the ruby-pcp-client gem
   # https://github.com/puppetlabs/ruby-pcp-client
   em = Thread.new { EM.run }
```

Comment:
Keeping a handle to this thread isn't really needed, unless you want to .join it after calling EM.stop.